builtin/settingtypes.txt

```diff
@@ -1062,6 +1062,13 @@ max_packets_per_iteration (Max. packets per iteration) int 1024
 #     (levels 1-3 use Zlib's "fast" method, 4-9 use the normal method)
 map_compression_level_net (Map Compression Level for Network Transfer) int -1 -1 9
 
+#	 When this is enbled, all item metadata will be sent to all clients. This
+# 	 might be useful to allow for local map saving, but it also increases
+# 	 lag, and malicious players could abuse it to crash the server or prevent
+# 	 other players from logging in by making the server crash automatically
+#	 when they log in.
```

Comment:
1. Wrong indents (should be all space)
2. Description outdated, since it's enabled by default for map saving
3. How could they crash the server? First that should be fixed, and second it does not belong into a setting documentation.

Suggestion:
```
Sends all item metadata fields to clients. Otherwise the server will truncate
irrelevant data per-client to reduce load and network traffic.
Note: Set to "true" to allow local map saving on clients prior to 5.5.0-dev.
```

Reply:
>     3. How could they crash the server?

https://github.com/minetest/minetest/blob/a21402b38faab484195224205ef0bbd112f72162/src/network/connection.cpp#L1017

Reply:
I changed it to your suggestion. The fact that the server can crash when trying to send packets that are too big is a different issue and should probably be solved by another PR.